core/src/main/res/values-qq/strings.xml

```diff
@@ -18,8 +18,8 @@
   <string name="menu_random_article">{{Identical|Random article}}</string>
```

Comment:
Is this metadata file necessary to be added?
If it is we should add `tools:ignore="all"` to it.
If it isn't I would look to delete/stop producing it

Reply:
Its highly recommended that the qqq file is retained. This is the message documentation file. 

You can read more about here - https://translatewiki.net/wiki/Translating:Localisation_for_developers#Message_documentation

The idea is that when you add a new string in English, you add some documentation in qq file to provide translators some context regarding the string.

I think we can add `tools:ignore="all"` part to the resources tag, will discuss with Niklas.

Reply:
Then I should document that on our side. I will create a ticket

Reply:
https://github.com/kiwix/kiwix-android/issues/1759 ticket added